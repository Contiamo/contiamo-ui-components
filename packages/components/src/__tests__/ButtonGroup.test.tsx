import * as React from "react"
import { render, mount } from "enzyme"

import { ButtonGroup as ThemelessButtonGroup, Button } from "../index"
import wrapDefaultTheme from "../utils/wrap-default-theme"

const ButtonGroup = wrapDefaultTheme(ThemelessButtonGroup)

describe("ButtonGroup Component", () => {
  it("Should initialize properly", () => {
    expect(
      render(
        <ButtonGroup>
          <Button>Hello</Button>
        </ButtonGroup>,
      ),
    ).toMatchSnapshot()
  })
})
