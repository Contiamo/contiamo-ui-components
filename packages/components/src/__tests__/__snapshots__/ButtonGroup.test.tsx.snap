// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonGroup Component Should initialize properly 1`] = `
<div
  class="css-1h5oz0d-buttongroup"
>
  <button
    class="css-1iqekuh-button"
  >
    Hello
  </button>
</div>
`;
